packages/core/src/Engine.ts

```diff
@@ -652,4 +657,6 @@ export interface EngineConfiguration {
   physics?: IPhysics;
   /** Color space. */
   colorSpace?: ColorSpace;
+  /** ShaderLab */
```

Comment:
Pay attention to the annotation specification:[shaderLab.initialize(](https://tsdoc.org/)